workspaces/leetcode-api/schema.graphql

```diff
@@ -9631,9 +9616,7 @@ type Query {
     numPerPage: Int
     pageNo: Int
   ): PagifiedOfficialSolutionFeedbackDetailNode
-  omFavoriteList(
-    data: OMFavoriteListInput! = { skip: 0, limit: 20, sort_dsc: true }
-  ): OMFavoriteListNode!
+  omFavoriteList(data: OMFavoriteListInput!): OMFavoriteListNode!
```

Comment:
The new approach fails to print non-scalar default values, but other than that it's the same output as the old schema. Will bring this back in another PR.